src/services/PingPongService.ts

```diff
@@ -66,20 +54,45 @@ export default class PingPongService {
               eip712.pingpong.Pong,
               {
                 serviceProvider: utils.arrayify(facilityId),
-                loc: LatLng.toBinary(loc),
+                loc: utils.toUtf8Bytes(h3Index),
+                timestamp: msg.timestamp,
```

Comment:
Not so sure about the logic here, I think we should just timestamp it with the present time on the way out prior to being signed. Given that this references `msg`, if the passed in `msg` doesn't have a `msg.timestamp` specified, when EIP-712 attempts to sign, this will fail (with a `seconds` / `nanos` missing). 

Therefore, I think when we send the reply, that we should send the present time in `seconds` and `nanos`.